client/resolvers.go

```diff
@@ -0,0 +1,24 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/plaid/plaid-go/v10/plaid"
+	"github.com/thoas/go-funk"
+)
+
+func ResolveNullableTime(path string) schema.ColumnResolver {
+	return func(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+		data := funk.Get(resource.Item, path)
+		if data == nil {
+			return nil
+		}
+		ts, ok := data.(plaid.NullableTime)
+		if !ok {
+			return fmt.Errorf("unexpected type, wanted \"*plaid.NullableTime\", have \"%T\"", data)
+		}
+		return resource.Set(c.Name, ts.Get())
```

Comment:
This is a workaround to an issue in `NullableTime` not checking for `nil` when unmarshalling:
https://github.com/plaid/plaid-go/blob/12412d452b3a00300ae0f1d58e61a0b43f55e5f9/plaid/utils.go#L327